src/Services/Masa.Scheduler.Services.Server/Domain/EventHandler/NotifyTaskRunResultDomainEventHandler.cs

```diff
@@ -34,8 +36,8 @@ public async Task NotifyTaskRunResultAsync(NotifyTaskRunResultDomainEvent @event
             case TaskRunStatus.Success:
                 message = "Task run success";
                 break;
-            case TaskRunStatus.Stop:
-                message = "Task run stop";
+            case TaskRunStatus.Timeout:
```

Comment:
change to :
var message = XXX switch
{
  TaskRunStatus.Success=>"",
TaskRunStatus.Timeout=>"",
}